File: SDKs/beezup.com/2.0/go/pkg/models/shared/publicationtypeenum.go
package shared

type PublicationTypeEnum string

const (
	PublicationTypeEnumPublishProducts PublicationTypeEnum = "PublishProducts"
	PublicationTypeEnumPublishOffers   PublicationTypeEnum = "PublishOffers"
	PublicationTypeEnumUnpublish       PublicationTypeEnum = "Unpublish"
)

File: SDKs/beezup.com/2.0/java/lib/src/main/java/openapisdk/models/shared/PublicationTypeEnum.java
package openapisdk.models.shared;


public enum PublicationTypeEnum {
    PUBLISH_PRODUCTS("PublishProducts"),
    PUBLISH_OFFERS("PublishOffers"),
    UNPUBLISH("Unpublish");

    public final String value;

    private PublicationTypeEnum(String value) {
        this.value = value;
    }
}
